dashboards/jetstream/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+sample_dashboards:
+  -
+    category: Jetstream
+    id: jetstream-prometheus
+    display_name: Jetstream Prometheus Overview
+    description: "This dashboard has charts displaying request rate, latency, queue sizes and other metrics for Jetstream inference."
+    related_integrations:
+      - id: jetstream
```

Comment:
FYI dashboard and integration will deploy separately

Reply:
Ack. I don't think it's a big deal because:
- the integration is being added as HIDDEN -- I'll wait for both to propagate fully, validate it internally and publish the corresponding docs before we mark it as GA
- I believe our FE code fails gracefully if either deploys before the other